eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteAuditDb.scala

```diff
@@ -87,7 +86,7 @@ class SqliteAuditDb(sqlite: Connection) extends AuditDb with Logging {
       statement.setLong(3, e.localBalanceMsat)
       statement.setLong(4, e.commitments.commitInput.txOut.amount.toLong)
       statement.setLong(5, e.commitments.remoteParams.channelReserveSatoshis) // remote decides what our reserve should be
-      statement.setLong(6, Platform.currentTime)
+      statement.setLong(6, Platform.currentTime.milliseconds.toSeconds)
```

Comment:
This should be `.toMilliseconds`

Reply:
Okay so audit DB unfortunately has milliseconds timestamp in it, commit 294cb0a  makes sure we still store then in that time unit but when the objects are instantiated we convert the millis to actual seconds.